mirlib/dio.c

```diff
@@ -341,6 +341,7 @@ void dreaddir_c(char *contxt,char *path,int length)
     Strcpy(path,dp->d_name);
     Strcpy(npath,d->path);    Strcat(npath,path);
     (void)stat(npath,&buf);
-    if(S_IFDIR & buf.st_mode)Strcat(path,"/");
+    if(S_ISDIR(buf.st_mode))
+      Strcat(path,"/");
```

Comment:
Is this coding style allowed in `casacore`? I mean, having an `if` statement on multiple lines without using curly braces?

Reply:
It's consistent with the rest of the file, that's about the code style we're aiming for. But I added curly braces here.